src/components/modal/ModalStepper.tsx

```diff
@@ -10,7 +10,7 @@ interface Props {
 export const ModalStepper = ({ labels }: Props) => {
   const { stepIndex } = useModalContext() || {};
 
-  if (!stepIndex) return null;
+  if (stepIndex === null) return null;
```

Comment:
stepIndex però può essere anche `undefined`

Reply:
Se ti fa diventare matto questo, context, modifica il type del `ContextProps` in

```typescript
type ContextProps = {
onClose?: () => any
stepIndex?: number
}
```

Reply:
In realtà controllando che non sia `null || undefined` funge